README.md

```diff
@@ -50,6 +50,15 @@ module ExampleApplication
 end
 ```
 
+The result object can be destructed to easily grab the value and errors.
```

Comment:
sp. -> `destructured`

Consider: "You can use destructuring assignment to easily grab the value and the errors"